textprocessors.h

```diff
@@ -16,6 +17,6 @@ void lowercase_letters(char *text);
 void leet_case(char *text);
 void capitalize_letters(char *text);
 void capitalize_letters(string &text); 
-void search_replace(char *text, char *search, char *replace);
+void search_replace(string &text, string &search, string &replace);
```

Comment:
```suggestion
void search_replace(string &text, const string search, const string replace);
```
Update signature to match `textprocessors.cpp`